garrysmod/lua/vgui/dcolormixer.lua

```diff
@@ -307,7 +307,7 @@ function PANEL:UpdateConVar( strName, strKey, color )
 	if ( !strName ) then return end
 	local col = color[strKey]
 
-	RunConsoleCommand( strName, tostring( col ) )
+	RunConsoleCommand( strName, col:ToString() )

```

Comment:
Yes, an alternative / better name would be better suited as ToString seems to confuse rather than say what it should be such as Show / Serialize / etc.... I was going to use ToSafeString but went with ToString but I am thinking of renaming this before this is accepted...